### PR TITLE
MULTIARCH-5150: Pass IAM endpoint override to PowerVS CloudProviderConfig

### PR DESCRIPTION
**Pull request by miyamotoh**
Signed-off-by: Hiro Miyamoto <miyamotoh@us.ibm.com>

### CODE REVIEW

File: pkg/asset/manifests/powervs/cloudproviderconfig.go [hunk 1]
```diff
@@ -88,4 +103,5 @@ powerVSCloudInstanceID = {{.Provider.PowerVSCloudInstanceID}}
 powerVSCloudInstanceName = {{.Provider.PowerVSCloudInstanceName}}
 powerVSRegion = {{.Provider.PowerVSRegion}}
 powerVSZone = {{.Provider.PowerVSZone}}
+iamEndpointOverride = {{.Provider.IamEndpointOverride}}
```
[the remark this answers is not in the document]
Comment (mjturek) [reply pkg/asset/manifests/powervs/cloudproviderconfig.go:106]: Let's try the CI job to be safe

**Review by r4f4 — COMMENTED** [review]
/approve

**Comment by mjturek** [comment]
/retitle MULTIARCH-5150: Pass IAM endpoint override to PowerVS CloudProviderConfig

**Pull request merged**